internal/controllers/machine/drain/filters.go

```diff
@@ -62,6 +62,17 @@ func (l *PodDeleteList) Pods() []*corev1.Pod {
 	return pods
 }
 
+// IgnoredPods returns a list of Pods that have to be ignored before the Node can be considered completely drained.
```

Comment:
nit: let's add a quick not about which pods we expect to fit into this group: DaemonSet, StaticPods, (in future pods with skip drain). Something else?

Reply:
Its a bit more complex.
This list also depends on the drainer's configuration, because if `SkipWaitForDeleteTimeoutSeconds` is set, also deleted pods may be part of that list.

Note: in case of waiting for volumes to detach: this is never the case because `SkipWaitForDeleteTimeoutSeconds` is not set.

Reply:
Added, PTAL :-)